common/changes/@itwin/property-grid-react/tree-widget-double-click-zoom_2023-12-05-15-49.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Resolve rush install warnings.",
```

```suggestion
      "comment": "",
```

these are supposed to be user facing